Doc/Tutorial/chapter_seq_annot.tex

```diff
@@ -132,21 +132,19 @@ \subsection{SeqRecord objects from FASTA files}
 used to loop over all the records in a file as \verb|SeqRecord| objects. The \verb|Bio.SeqIO| module
 has a sister function for use on files which contain just one record which we'll use here (see Chapter~\ref{chapter:seqio} for details):
 
-%TODO - line wrapping for doctest?
```

Comment:
How bad does this look in the PDF output? I forget the timing, maybe switching to minted for the code blocks helps?